backend/protocol_rpc/endpoints.py

```diff
@@ -202,8 +208,9 @@ def update_validator(
     model: str,
     config: json,
 ) -> dict:
-    if not accounts_manager.is_valid_address(validator_address):
-        raise InvalidAddressError(validator_address)
+    # Remove validation while adding migration to update the db address
+    # if not accounts_manager.is_valid_address(validator_address):
+    #     raise InvalidAddressError(validator_address)
```

Comment:
should we revert these comments?

Reply:
I added them now to allow the validators update. Maybe my comment is not accurate but I meant that I didn't want to add the validation until we migrate the validators address in the DB to be correct